docs/source/configuration.rst

```diff
@@ -1,10 +1,17 @@
-Configuring bravado
-======================
+Configuration
+=============
 
 You can configure certain behaviours when creating a ``SwaggerClient``.
 
+The full documentation for each config key is in `bravado-core <http://bravado-core.readthedocs.org/en/latest/config.html>`_.
+
 .. code-block:: python
 
+    from bravado_core.formatter import SwaggerFormat
```

Comment:
I don't think we need to expose `bravado-core` interface to a `bravado` user. I would like to have it imported in `bravado.client` or `bravado.formatter` module, and let the user import that instead.